server/src/instant/db/permissioned_transaction.clj

```diff
@@ -138,7 +138,8 @@
 
 (defn object-view-check-fn [{:keys [etype program data] :as _check}
                             {:keys [current-user] :as ctx}]
-  (let [{:keys [original]} data]
+  (let [{:keys [original]} data
+        etype-str (name etype)]
```

Comment:
Why did we make it into a keyword in the first place? Looks like we do that below in `object-view-check`?

Reply:
Good q! We did this previously on the [`check`](https://github.com/instantdb/instant/blob/794c34a1f17d71537c91b12cccf38dd4974faab1/server/src/instant/db/permissioned_transaction.clj) . I had assumed that some other caller was relying on the etype being a keyword, but I'll do a check and remove the keywordiness if all looks good